src/client/pages/OfferNew/Checkout/index.tsx

```diff
@@ -133,12 +133,12 @@ export type CheckoutProps = {
   refetch: () => Promise<void>
 }
 
-export const Checkout: React.FC<CheckoutProps> = ({
+export const Checkout = ({
```

Comment:
nit: don't really understand why this change is preferable

Reply:
[This Github issue](https://github.com/facebook/create-react-app/pull/8177) was mentioned on a article that I've read about [writing react components with Typescript](https://kentcdodds.com/blog/how-to-write-a-react-component-in-typescript). It's not a big deal. In fact that change shouldn't have been committed. 😬